README.md

```diff
@@ -3,7 +3,7 @@
 A data type for optional values.
 
 ## Table of Contents
-
+* [Installation](#installation)
```

Comment:
Looks like the blank line got lost. Also, if `Spago` is going to be its own heading, we should add it to the table of contents:

```suggestion

* [Installation](#installation)
    * [Spago](#spago)
```